public/app/plugins/datasource/prometheus/dashboards/grafana_stats.json

```diff
@@ -1,8 +1,8 @@
 {
   "__inputs": [
     {
-      "name": "DS_PROMETHEUS",
-      "label": "Prometheus",
+      "name": "DS_GDEV-PROMETHEUS",
+      "label": "gdev-prometheus",
       "description": "",
```

Comment:
I am wondering why 
```
      "name": "DS_PROMETHEUS",
      "label": "Prometheus",
```

was changed to

```
      "name": "DS_GDEV-PROMETHEUS",
      "label": "gdev-prometheus",
```

And in the other 2 files it was changed from `DS_NAME` to `DS_GDEV-PROMETHEUS`. 

I am sure it has no effect on functionality, but it seems the naming was more generic. So I was curious why is this.

Reply:
Hmm that's interesting. Maybe it had to do with the datasource that i exported the dashboards from is called gdev-prometheus?

Reply:
Yeah, I tried to export it and it kept the name of my ds as well. 🙂 This is super minor thing that doesn't affect functionality, so I'll leave it up to you if you want to rename these inputs/variables to use generic name or keep it like this. Let me know if you want to keep it and I'll approve. 🙂

Reply:
I think i'll go ahead and change them to a generic name instead 👍